src/etc/cargo.bashcomp.sh

```diff
@@ -69,7 +69,7 @@ _cargo()
 	local opt__login="$opt_common $opt_lock --registry"
 	local opt__metadata="$opt_common $opt_feat $opt_mani $opt_lock --format-version=1 --no-deps --filter-platform"
 	local opt__new="$opt_common $opt_lock --vcs --bin --lib --name --edition --registry"
-	local opt__owner="$opt_common $opt_lock -a --add -r --remove -l --list --index --token --registry"
+	local opt__owner="$opt_common $opt_lock -a --add add -r --remove remove -l --list list --index --token --registry"
```

Comment:
See `cargo report future-incompat` for an example of how to do completions for subcommands

Reply:
It has been modified.